code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -248,7 +248,7 @@
 	heatmod = 1.5
 	info_text = "As a <span class='danger'>Wooden Golem</span>, you have plant-like traits: you take damage from extreme temperatures, can be set on fire, and have lower armor than a normal golem. You regenerate when in the light and wither in the darkness."
 	prefix = "Wooden"
-	special_names = list("Tomato", "Potato", "Broccoli", "Carrot", "Ambrosia", "Pumpkin", "Ivy", "Kudzu", "Banana", "Moss", "Flower", "Bloom", "Root", "Bark", "Glowshroom", "Petal", "Leaf", "Venus", "Sprout","Cocoa", "Strawberry", "Citrus", "Oak", "Cactus", "Pepper", "Juniper")
+	special_names = list"Bark" "Oak" "Sap" "Twig" "Branch" "Maple" "Birch" "Elm" "Basswood" "Cottonwood" "Larch" "Aspen")

```

Comment:
Put a `(` before `"Bark"`.

Reply:
thanks

Reply:
add the following;

- Ash
- Beech
- Buckeye
- Ceder
- Chestnut
- Cypress
- Fir
- Hawthorn
- Hazel
- Hickory
- Ironwood
- Juniper
- Leaf
- Mangrove
- Palm
- Pawpaw
- Pine
- Poplar
- Redbud
- Redwood
- Sassafras
- Spruce
- Sumac
- Trunk
- Walnut
- Yew